### DISCUSSION TITLE
Add the Windows 7 details pane to ExplorerPatcher

### DISCUSSION THREAD

**Discussion by EJSnow** [valinet/ExplorerPatcher#2060] [Ideas]
The title says it all, I guess. The Windows 7 details pane is really nice in that it allows you to also have the preview pane enabled, and it's kinda annoying to have to keep toggling back and forth, because sometimes I want to see details of files, and sometimes I want a preview of the file's contents, and with the Windows 7 details pane, I could do that without toggling back and forth. But on the Windows 10 interface, the preview pane and details pane share the same space to the right of the folder's contents in Explorer. And enabling one disables the other. So I would like for this to be added to ExplorerPatcher. Probably, you could just have setting the control interface for Explorer to the Windows 7 interface also enable the Windows 7 details pane as well. ~~I will completely understand if you choose not to implement this though.~~

**Comment by pyrates999** [comment]
Only if the code for this still exists in windows 10.

**Reply by EJSnow** [reply]
It still exists as the Programs and Features control panel uses the Windows 7 details pane to display more details about the selected program.

**Comment by vippium** [comment]
@Amrsatrio Is it possible to recover that from P&F page? This is a demanded feature.

**Reply by Amrsatrio** [reply]
Just use OldNewExplorer. I don't know how it works, I had a stroke when  reading its decompiled code.

**Reply by EJSnow** [reply]
Here's the thing. OldNewExplorer doesn't have the removal of context menu skinning in Explorer on Windows 10. ExplorerPatcher does though. (Oh, how I wish Microsoft hadn't removed the ability to just remove the skinning by just setting a registry key.)

**Comment by mr-september** [comment]
I think the demand for this (or similar) feature is going to go up because [MSFT just made it even worse](https://answers.microsoft.com/en-us/windows/forum/all/windows-11s-new-file-explorer-details-pane/99479aa6-0268-4fc9-bd48-fa10acc28f41).

LesFarch's EP suggestion "Windows 11 Command Bar (classic Address Bar)" no longer works for me. "Win 10 Ribbon UI" works but that loses the tabs functionality.

I would love for details pane mods to become a dedicated, clearly named option as well.

**Reply by vippium** [reply]
Yup...this is going on. And I think that @Amrsatrio is aware of it. Maybe fixed with nxt pre-release

**Reply by EJSnow** [reply]
You can get the classic address bar in the Windows 11 File Explorer back without losing tabs/the rest of Explorer's redesign with a [Windhawk](https://windhawk.net) mod I believe.

**Comment by Tarek-Hasan** [comment]
[Windhawk](https://windhawk.net), windows customizer has this feature in [Aerexplorer](https://windhawk.net/mods/aerexplorer) mod. Here is the [source code](https://github.com/ramensoftware/windhawk-mods/blob/main/mods/aerexplorer.wh.cpp) of Aerexplorer mod. Please, check this out if you're interested in adding the feature.

**Reply by EJSnow** [reply]
Yeah, I've actually moved to Aerexplorer and I'm using it in my Windows 7 theme on Windows 10. So honestly, idrc about having this functionality in ExplorerPatcher anymore because everything I used in ExplorerPatcher is already available across a few Windhawk mods.

**Comment by Tarek-Hasan** [comment]
Love to have this feature.
